### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Safiya-a**

### CODE REVIEW

**Review by Augustine129 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,10 +44,16 @@ Below are the two designs that have been created, these will then be placed in a
 
 https://docs.google.com/forms/d/e/1FAIpQLSc7P46kY3y1qZsO-8r04ekQAPc0k2TGqmJHWf1DGV9p-H6hSg/viewform
 
+## UI Design Results
+
+As shown in the results above, the majority votes chose the second option. We chose to use a google form to capture the results as it was a simple way to recieve the feedback from the team. With this A/B testing method, we were able to provide the benefits of each sample. The second sample had a simpler and easy design so this is what we decided to choose.
```
Comment (kesh4207) [inline README.md:49]: Very well put Safiya!  I think it would be more simpler to do design 1.

**Review by kesh4207 — APPROVED** [review]
Yes, I agree!

File: README.md (hunk 1 above)
Comment (Safiya-a) [reply README.md:49]: Thank you Keseana!

**Review by jamesspencer19 — APPROVED** [review]
Looks good.

**Pull request merged**